### PR TITLE
refactor water order feature code

### PR DESCRIPTION
**Pull request by lazycoder9**

### CODE REVIEW

File: features/waterOrdering/index.js [hunk 1]
```diff
@@ -5,36 +5,23 @@ const queries = require('./queries.js')
...
-        await bot.reply(message, { text: randomArrayItem(tooMuchOrdersReplies(lastOrderTime))});
-      }
-    },
-  )
-}
+const DEFAULT_WATER_ORDER_INTERVAL = 5 * 24 * 60 * 60 // 5 days in seconds
 
-const tooMuchOrdersReplies = (lastOrderTime) => (
-  replies.tooMuchOrdersError.map((reply) => (
+const tooMuchOrdersReplies = lastOrderTime => (
```
Comment (AleksSenkou) [inline features/waterOrdering/index.js:10]: thoughts about adding the linter to Mona?

File: features/waterOrdering/index.js [hunk 2]
```diff
@@ -78,13 +57,36 @@ const passedEnoughTimeFromLastOrder = async (robot) => {
...
 
   if (lastWaterOrder) {
-    const currentDate = new Date();
+    const currentDate = new Date()
 
-    if ((currentDate - lastWaterOrder) / 1000 >
-      (process.env.WATER_ORDER_MIN_INTERVAL || DEFAULT_WATER_ORDER_INTERVAL)) {
-      return (true);
+    if ((currentDate - lastWaterOrder) / 1000
+      > (process.env.WATER_ORDER_MIN_INTERVAL || DEFAULT_WATER_ORDER_INTERVAL)) {
```
Comment (AleksSenkou) [inline features/waterOrdering/index.js]: can this be split into several functions for better readability?

Comment (lazycoder9) [reply features/waterOrdering/index.js]: @AleksSenkou yeah, it is just a start of refactoring. Created PR so early to check if our Github  Actions is working

**Review by npupko — APPROVED** [review]
@lazycoder9 LGTM

**Review by AleksSenkou — APPROVED** [review]
LGTM

**Pull request merged**